wolfssl/ssl.h

```diff
@@ -3505,7 +3511,7 @@ WOLFSSL_API void* wolfSSL_GetEncryptKeysCtx(WOLFSSL* ssl);
 
 typedef int (*CallbackTlsFinished)(WOLFSSL* ssl,
                             const byte *side,
-                            const byte *handshake_hash,
+                            const byte *handshake_hash, word32 hashSz,
```

Comment:
I think we can take this API breakage... `CallbackTlsFinished` But need to confirm.

Reply:
Thanks!! Yes. We talked about this over internal chat. Is there someone else I need to check with? The change does make sense as the hash size is not a constant.